src/components/ImportCombinePopUp.vue

```diff
@@ -66,9 +80,15 @@ export default {
       if (!match) {
         this.showError = true
       } else {
+        let newIds = match[0].split(',')
+        let currentIds = this.currentSelectedIds.split(',')
+        if (this.combineWithCurrentSelectedIds) {
+          newIds = _.uniq([...currentIds, ...newIds].filter(x => x))
```

Comment:
What does filter x=> x do?

Reply:
`x => x` is a function which returns the given parameter. Also known as an [identity](https://en.wikipedia.org/wiki/Identity_function) function.

It is used here to filter out `falsy` values.
`filter` loops over all items in an array. It accepts a function, which should return `true` or `false`.  When `true` the array item is kept, when `false`, the array item is discarded.

Imagine the array `["", "one", "two"]`. 
When looped over with `filter` (`["", "one", "two"].filter(x => x)`), the first time, the function given to filter (`x => x`) gets `""` as its argument. The function returns it. `filter()` will [coerce](https://2ality.com/2019/10/type-coercion.html) it into a Boolean. `Boolean("")` will become `false`, thus, the fist item of the array is discarded.
The second and third time, the result of `Boolean("one")` and `Boolean("two")` will be `true`, thus, these values will remain.

So: `console.log(["", "one", "two"].filter(x => x)) // ["one", "two"]`

The remaining array is passed to Lodash's `uniq` function which will remove all duplicates. This results in an array without falsy, unique values

Reply:
We might have an id: 0. Will that scenario be removed?

Reply:
it will

Reply:
No, it will not. the id's will always be strings, so the value will be: `"0"`.  That is not falsy